Dockerfile

```
@@ -16,8 +17,22 @@ RUN apt-get update && apt-get install -y \
     # Media processing
     ffmpeg \
     # Utilities
-    curl unzip \
-    && rm -rf /var/lib/apt/lists/*
+    curl unzip
+
+RUN if [ "$(dpkg --print-architecture)" = "amd64" ]; then \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)